File: Terrazas-Chavez-Jose-Alexis/Practicas/Python/practica_0.py
'''
Programa que recibe argumentos de entrada e imprime el número de ellos
'''
import sys

def main():
        argc = len(sys.argv)  # Cuenta el número de argumentos
        print (f"argc = {argc}")

        if argc == 1:
                return sys.exit(0) 
        else:
                return sys.exit(argc)  # Retorna el número de argumentos si hay más de uno

if __name__ == "__main__":
        main()

File: Terrazas-Chavez-Jose-Alexis/Practicas/C/practica_0.c
/*
Programa que recibe argumentos de entrada e imprime el número de ellos
Es necesario compilar el programa de la siguiente manera 'gcc -o practica_0 practica_0.c'
ppsteriormente se ejecuta de la siguiente manera ./practica_0 ls echo
*/
#include <stdio.h>

int main(int argc, char *argv[]) {
    printf("argc = %d\n", argc);

    if (argc == 1) {
        return 0;
    } else {
        return argc;
    }
}
